packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/opensignin.md

```diff
@@ -1,26 +1,44 @@
-Sign the user in by updating the `signIn` function of `ContentView.swift`:
+Update the `signIn` function of `ContentView.swift` with the code to sign in the user:
 
 ```swift
 func signIn() {
-  guard let rootViewController = UIApplication.shared.rootViewController else {
-    showError(title: "Unable to Sign In",
-    message: "An unkown error occured. Try restarting the app.")
-    return
-  }
+   Task {
+      do {
+         if let token = try await WebAuthentication.shared?.signIn(from: nil)
+         {
+            try Credential.store(token)
+            updateStatus("Signed in", infoText: "", signedInStatus: true)
+         }
+      } catch {
+         // Handle errors
+         showError(title: "Unable to Sign In", error: error)
+      }
+   }
+}
+```
+
+The app hands the sign-in flow to the Okta SDK. The SDK presents the sign-in web view and waits for a result. If the sign-in is succesful, the app first stores the token using `Credential.store(_)`, and then updates the user interface by setting the `signedIn` state variable to `true`.
```

Comment:
sign-in shouldn't be used as a noun acc2 the style guide.

```suggestion
The app hands the sign-in flow to the Okta SDK. The SDK presents the sign-in web view and waits for a result. If the sign-in attempt is successful, the app first stores the token using `Credential.store(_)`, and then updates the user interface by setting the `signedIn` state variable to `true`.
```

Reply:
Done. In the next commit.